src/pages/projects/form/ProjectParticipants.tsx

```diff
@@ -30,24 +30,16 @@ export const ProjectParticipants = () => {
 
   const toggleOpenAddProjectParticipantView = () => setOpenAddProjectParticipantView(!openAddProjectParticipantView);
 
-  const removeProjectParticipant = (name: string, remove: (index: number) => void, contributorIndex: number) => {
-    const projectManagerRole = findProjectManagerRole(values.contributors[contributorIndex]);
-
-    // Contributor also has Project manager role: Only delete all the others
-    if (projectManagerRole) {
-      const newContributors = replaceRolesOnContributor(values.contributors, contributorIndex, [projectManagerRole]);
-      setFieldValue(name, newContributors);
-    } else {
-      // Does not have project manager role: Delete the whole contributor
-      remove(contributorIndex);
-    }
+  const onRemoveContributor = (name: string, contributorIndex: number) => {
```

Comment:
samme som tidligere kommentar her også. Jeg bruker metodesignaturen til å forstå hva den gjør, og her tolker jeg det som at `name` er navnet på bidragsyteren som skal fjernes. Når jeg leser selve funksjonen ser jeg det ikke er tilfellet da. Stemmer for å enten legge denne funksjonen inline, eller rename param til `fieldName` e.l.